recipes/bleties/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - htslib=1.9
     - biopython=1.76
     - scipy=1.5.0
-    - muscle>=3.8.1551
+    - muscle=3.8.1551
```

Comment:
shouldn't this be `<=`

Reply:
No, 3.8.1551 was the version used for testing. Lower versions were not tested. Higher versions break version check as described above.